src/jobs/facebookPost/index.js

```diff
@@ -5,7 +5,7 @@ const Bugsnag = require('@bugsnag/js')
 const BugsnagPluginExpress = require('@bugsnag/plugin-express')
 
 Bugsnag.start({
```

Comment:
@ShivaAryal do we need `Bugsnag.start` in multiple places? In fact, we should remove Bugsnag.